src/views/blog/BlogIndex.vue

```diff
@@ -1,40 +1,190 @@
 <template>
   <div>
-    <div id="blogPostList">
-      <blog-post-component
-      v-for="post in currentBlogPosts"
-      :key="post.id"
-      :post="post">
-      </blog-post-component>
+    <div class="featured-section">
+      <div class="featured-image">
+        <img :src="featuredPost.coverImageURL">
+      </div>
+      <div class="featured-content">
+        <div class="tags-container">
+          <div v-for="tag in featuredPost.tags" class="tag">
+            {{tag}}
+          </div>
+        </div>
+        <div class="title">
+          Featured: {{featuredPost.title}}
+        </div>
+        <div class="half-divider"/>
+        <div class="sub-title">
+          {{featuredPost.subTitle}}
+        </div>
+        <div class="author-info">
+          {{featuredPost.author.firstName}} {{featuredPost.author.lastName}} {{featuredPost.createdAt}}
+        </div>
+        <div class="sample-description">
+          {{truncateFeature(featuredPost.body)}}
+        </div>
+      </div>
+    </div>
+    <div :class="getClassName(post)" v-for="post in otherPosts" v-bind:key="post.id">
+      <div class="post-image-small">
+        <img :src="post.coverImageURL">
+      </div>
+      <div class="post-content">
+        <div class="tags-container">
+          <div v-for="tag in post.tags" class="tag">
+            {{tag}}
+          </div>
+        </div>
+        <div class="title-small">
+          Featured: {{post.title}}
+        </div>
+        <div class="sub-title">
+          {{post.subTitle}}
+        </div>
+        <div class="author-info">
+          {{post.author.firstName}} {{post.author.lastName}} {{post.createdAt}}
+        </div>
+        <div class="sample-description">
+          {{truncateSmall(post.body)}}
+        </div>
+      </div>
     </div>
   </div>
 </template>
 
 <script>
-import BlogPostComponent from '../../components/BlogPostComponent.vue';
+import _ from 'lodash'
+import { mapGetters } from 'vuex'
 
 export default {
-  components: {
-    BlogPostComponent
-  },
-
   computed: {
-    currentBlogPosts() {
-      return this.$store.getters['blog/getPageOfPosts'];
+    ...mapGetters({
+      latestPosts: 'blog/getLatestPosts'
+    }),
+    featuredPost () {
+      return this.latestPosts[0]
     },
-
-    numberOfPages() {
-      return this.$store.getters['blog/getNumberOfPages'];
+    otherPosts () {
+      return this.latestPosts.slice(1, 99) // Will eventually be getting this information from the API in pages of 10 posts each. Eventually will not need to be sliced and will just start with 10
     }
   },
-
-  mounted() {
```

Comment:
In general, we don't need the `mounted` hook here. We should use `data` and `computed` for initial values in a vue component. An example would be:

```
computed: {
  pageOnLoad () {
    return this.$route.query.page
  }
}
```
And we can do more work from there once the page has loaded. here would could also dispatch an action that fetches the initial 10 blog posts from the backend and loads them up.